crypto/monero/address_test.go

```diff
@@ -27,3 +28,15 @@ func TestValidateAddress(t *testing.T) {
 	err = ValidateAddress("fake", common.Mainnet)
 	require.True(t, errors.Is(err, errInvalidAddressLength))
 }
+
```

Comment:
it would be good to add some test vectors to this file that contain real monero addresses to ensure we can decode them properly